.github/CODEOWNERS

```diff
@@ -1,2 +1,11 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/machine-learning
\ No newline at end of file
+*   @opensearch-project/machine-learning
```

Comment:
Should we remove this team ?

Reply:
Seems this PR removed team https://github.com/opensearch-project/sql-cli/pull/12/files